sdk/maps/azure-maps-search/swagger/README.md

```diff
@@ -0,0 +1,47 @@
+# Azure Maps Search for Python
```

Comment:
Why do we need `swagger` directory here?

Reply:
We can generate from Swagger directory from this repo.